inst/include/RcppSimdJson/deserialize/dataframe.hpp

```diff
@@ -25,24 +25,23 @@ struct Column_Schema {
 template <Type_Policy type_policy, utils::Int64_R_Type int64_opt>
 inline auto diagnose_data_frame(simdjson::dom::array array) noexcept(RCPPSIMDJSON_NO_EXCEPTIONS)
```

Comment:
I don't know what RCPPSIMDJSON_NO_EXCEPTIONS does. 

The changes I made do not introduce exceptions, my only concern is that I do not otherwise ever address RCPPSIMDJSON_NO_EXCEPTIONS.

Reply:
See `RcppSimdJson/common.hpp` -- I think we accomodate different compilers (and local configs) here.  Default is exceptions enabled.